pytermgui/enums.py

```diff
@@ -16,8 +16,9 @@
 
 from __future__ import annotations
 
+from enum import IntEnum
+from enum import auto as _auto
```

Comment:
Not really sure if this is any better, but it is out of the scope of this PR.

Reply:
I only touched `ansi_interface` manually. All others are after `black` and `isort`'s usage

Reply:
Next time it would be better to only commit the file you are changing.